app/services/unsubscribe.rb

```diff
@@ -0,0 +1,27 @@
+module Unsubscribe
```

Comment:
Should we call this `UnsubscribeService`?

Reply:
I don't mind, @gpeng ?